### PR TITLE
feat: initial text

### PR DESCRIPTION
**Pull request by marvinmarnold**
Issue: https://github.com/NebraLtd/maker-starter-app/issues/11
Summary: Need to update assets so that app will have our branding and sensible text.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "MakerApp",
-  "displayName": "Maker App",
+  "name": "The Nebra Helium App",
```
Comment (marvinmarnold) [inline app.json:2]: @vpetersson is there a different official name?

File: src/locales/en.ts [hunk 2]
```diff
@@ -51,8 +51,8 @@ export default {
       create_account: 'Create an account on the Helium Network',
       login_with_helium: 'Already have a Helium Wallet?\nTap to get started.',
       subtitle:
-        'Lorem ipsum <b><errorText>$HNT</errorText></b> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
-      title: 'Welcome\nto Maker App',
+        'Use this app to onboard your miner for the first time and start earning HNT. <b>WARNING: You must also have the official Helium App installed in order to continue.</b>',
```
Comment (marvinmarnold) [inline src/locales/en.ts:54]: image

File: src/locales/en.ts [hunk 3]
```diff
@@ -167,6 +168,7 @@ export default {
           title: '[Placeholder]',
         },
       ],
+      // TODO
```
Comment (marvinmarnold) [inline src/locales/en.ts:171]: image

File: src/makers/example/hotspots.ts [hunk 4]
```diff
@@ -2,12 +2,14 @@ import HotspotIcon from './icon.svg'
 import { MakerHotspot } from '../hotspotMakerTypes'
 import ANTENNAS from './antennas'
 
+// TODO: variants
```
Comment (marvinmarnold) [inline src/makers/example/hotspots.ts:5]: @vpetersson @shawaj we must define all the miner types we support.

- Indoor (Raspberry Pi)
- Outdoor (Raspberry Pi)
- Indoor (ROCK Pi)

What other variants do we want to include in the app? What are the public facing names for these variants? The labels currently say something like "Nebra Helium Hotspot: NEBHNT-HHRK4-915", none of which makes sense to use in this context.

File: src/makers/example/hotspots.ts [hunk 5]
```diff
@@ -45,6 +47,7 @@ const ExampleHotspotQR = {
   onboardType: 'QR',
   translations: {
     en: {
+      // Different instructions
```
Comment (marvinmarnold) [inline src/makers/example/hotspots.ts:50]: I need to familiarize myself with this flow, not sure right now

File: src/makers/example/index.ts [hunk 6]
```diff
@@ -4,6 +4,6 @@ import hotspots from './hotspots'
 export default {
   antennas,
   hotspots,
-  id: 99999, // TODO: Get maker id from onboarding server
-  supportEmail: 'support@examplemaker.com',
+  id: 99999, // TODO: Get maker id from onboarding server after creation
```
Comment (marvinmarnold) [inline src/makers/example/index.ts:7]: @vpetersson after you run the node create-maker task, we need to update this with the corresponding ID

**Comment by marvinmarnold** [comment]
@posterzh is this the only change I need to revert to fix the name change? https://github.com/NebraLtd/maker-starter-app/pull/21/commits/0d2aba3f08e9c7ec0d8d57f4f0667e626f38b1ad

**Comment by posterzh** [comment]
> @posterzh is this the only change I need to revert to fix the name change? [0d2aba3](https://github.com/NebraLtd/maker-starter-app/commit/0d2aba3f08e9c7ec0d8d57f4f0667e626f38b1ad)

Correct.

**Comment by marvinmarnold** [comment]
@posterzh @Andrewsopoku just converted this to a real PR. We can still use the convo as a list of todo items, but I want to start getting things merged.

**Comment by marvinmarnold** [comment]
@posterzh @Andrewsopoku can you please review?

**Comment by marvinmarnold** [comment]
Thanks @Andrewsopoku I hadn't seen that README. I'll create some tickets accordingly.

**Comment by marvinmarnold** [comment]
@Andrewsopoku @posterzh  I've updated the PR so it only touches the text. I've updated the original ticket with all the specific UI details: https://github.com/NebraLtd/maker-starter-app/issues/11. Created additional tickets that you can both help with:

- https://github.com/nebraltd/maker-starter-app/issues/25
- https://github.com/nebraltd/maker-starter-app/issues/26

**Review by posterzh — APPROVED** [review]

**Pull request merged**